tests/cases/standalone/common/copy/copy_database_from_fs_parquet.result

```diff
@@ -0,0 +1,58 @@
+CREATE TABLE demo(host string, cpu double, memory double, ts TIMESTAMP time index);
+
+Affected Rows: 0
+
+INSERT INTO demo(host, cpu, memory, ts) values ('host1', 66.6, 1024, 1655276557000), ('host2', 88.8,  333.3, 1655276558000);
+
+Affected Rows: 2
+
+COPY DATABASE public TO '/tmp/demo/export/parquet/' WITH (FORMAT="parquet");
```

Comment:
Does this work on Windows?

Reply:
sqlness does not work on Windows. 

Path problem in Windows is addressed in this https://github.com/GreptimeTeam/docs/pull/794/files#diff-1a48dcb8961e2590022b5c248548d479e1a88e3efa895fec056943e749d902b4R135-R145